ac/ac-induction/src/index.js

```diff
@@ -46,8 +68,9 @@ const meta = {
 export default ({
   id: 'ac-induction',
   type: 'react-component',
-  meta,
   config,
+  meta,
   ActivityRunner,
+  merge,
```

Comment:
This should be a field call `mergeFunction`. If you don't give it a correct name it won't be used by the engine. did you make sure it works correctly? I get the following error, after a clean install on your branch, then creating an ac-induction and trying to run it. Also, same error when trying to see the preview. It seems to me it could be due to the non initialization of the reactive data.

![screenshot from 2017-08-11 15-13-18](https://user-images.githubusercontent.com/8818081/29214706-aa5ee93e-7ea8-11e7-87bb-fed26d371a33.png)